### PR TITLE
fix build error on loongarch64

### PR DESCRIPTION
**Pull request by zhangwenlong8911**

### CODE REVIEW

File: librz/debug/p/native/linux/reg/linux-loongarch64.h [hunk 1]
```diff
@@ -0,0 +1,52 @@
+// SPDX-FileCopyrightText: 2016 dromero <dromero.perez@hotmail.com>
```
Comment (Rot127) [inline librz/debug/p/native/linux/reg/linux-loongarch64.h]: Please set your copyright.

File: librz/debug/p/native/linux/reg/linux-loongarch64.h [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+// SPDX-License-Identifier: LGPL-3.0-only
+return strdup(
```
Comment (Rot127) [inline librz/debug/p/native/linux/reg/linux-loongarch64.h]: newline above please

**Review by Rot127 — CHANGES_REQUESTED** [review]

**Comment by Rot127** [comment]
Also, please run `clang-format` on the files.

File: librz/debug/p/native/linux/reg/linux-loongarch64.h (hunk 1 above)
Comment (wargio) [reply librz/debug/p/native/linux/reg/linux-loongarch64.h]: also fix the year

Comment (zhangwenlong8911) [reply librz/debug/p/native/linux/reg/linux-loongarch64.h]: I have updated the code. Thank you for your suggestion

**Review by XVilka — APPROVED** [review]

**Review by Rot127 — APPROVED** [review]

**Review by wargio — APPROVED** [review]

**Pull request merged**